src/SB/Core/gc/iModel.cpp

```diff
@@ -25,6 +30,28 @@ void* GetHierarchy(RpAtomic* frame)
     return unk_0[0];
 }
 
+
+void iModelInit()
+{
+    RwFrame* frame;
+    RwRGBAReal black;
+    void* tempVoid = 0;
+
+    for (S32 i = 0; i < 4; i++)
+    {
+        frame = (RwFrame*)RpLightCreate(i);
+        //black = frame;
```

Comment:
Please remember to take out commented code - it adds unnecessary noise when trying to debug functions to increase the match.